src/App.jsx

```diff
@@ -14,33 +16,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET_REVERSE = 'alphabetically_reverse';
+const SORT_FIELD_LENGTH_REVERSE = 'length_reverse';
+const SORT_FIELD_REVERSE = 'reverse';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField }) {
```

Comment:
pass reverse here as an additional boolean param

and append it later, if needed

```suggestion
function getPreparedGoods(goods, { sortField, reversed }) {
```